tensorbay/opendataset/UAVDT/loader.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+#
+# Copytright 2020 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name, missing-module-docstring
+
+import os
+from collections import defaultdict
+from typing import Any, Dict, Iterable, List, Tuple, Union
+
+from ...dataset import Data, Dataset
+from ...label import AttributeInfo, Classification, LabeledBox2D
+from ...utility import NameList
+from .._utility import glob
+
+_DATASET_NAME = "UAVDT"
+
+_MODES = ("train", "test")
+
+
+def UAVDT(path: str) -> Dataset:
+    """Dataloader of the `UAVDT`_ Dataset.
+
+    .. _UAVDT: https://sites.google.com/site/daviddo0323/projects/uavdt
+
+    The file structure of UAVDT looks like:
+
+        <path>
+            M_attr/
+                test/
+                    M0203_attr.txt
+                    ...
+                train/
+                    M0101_attr.txt
+                    ...
+            UAVDT_Benchmark_M/
+                M0101/
+                    img000001.jpg
+                    ...
+                ...
+            UAV-benchmark-MOTD_v1.0/
+                GT/
+                    M0101_gt_ignore.txt
+                    M0101_gt.txt
+                    M0101_gt_whole.txt
+                    ...
+
+    Arguments:
+        path: The root directory of the dataset.
+
+    Returns:
+        Loaded :class:`~tensorbay.dataset.dataset.Dataset` instance.
+
+    """
+    root_path = os.path.abspath(os.path.expanduser(path))
+
+    dataset = Dataset(_DATASET_NAME)
+    dataset.notes.is_continuous = True
+    dataset.load_catalog(os.path.join(os.path.dirname(__file__), "catalog.json"))
+
+    for mode in _MODES:
+        for label_path in glob(os.path.join(root_path, "M_attr", mode, "*.txt")):
+            sequence = os.path.basename(label_path).split("_")[0].strip()
+            segment = dataset.create_segment(f"{mode}-{sequence}")
+
+            sequence_attributes = _convert_attr_txt_to_dict(
```

Comment:
function named `_convert_attr_txt_to_dict` but inputs a label_path,
should the name be "extract" or "get" label from "file"?